classes/app.rb

```diff
@@ -70,4 +71,68 @@ def show_rentals_for_person(person_objects_array)
       end
     end
   end
+
+  def save_books_to_json_file
+    File.write('../dataFiles/books.json', JSON.generate(Book.all_instances))
+  end
+
+  def save_persons_to_json_file
+    File.write('../dataFiles/persons.json', JSON.generate(@persons))
+  end
+
+  def save_rentals_to_json_file
+    File.write('../dataFiles/rentals.json', JSON.generate(Rental.all_instances))
+  end
```

Comment:
- [ ] @hernandanielzamora @sameedmohsin28 and @dazehinn Great work implementing the changes requested however you have referenced your JSON files wrongly leading to the error below, instead of `'../dataFiles/books.json'` it should be `'./dataFiles/books.json'`, this also caused the data stored not to be read.
![image](https://github.com/sameedmohsin28/OOP-school-library/assets/61408860/318107b7-b973-4295-8d24-bd2c0d62aeef)

Reply:
Hey! @danmainah Thanks for your review, so the path should be the same for every file? And also, should it be the same when we want to read the files?